modules/nf-core/busco/main.nf

```diff
@@ -0,0 +1,84 @@
+process BUSCO {
```

Comment:
Similar to my comment below, only include it when it's actually used. Makes PRs smaller and focused and we don't have bunch of unused code around